### PR TITLE
[protocols] Update to 2.0

### PR DESCRIPTION
**Pull request by ffflorian**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/IonicaBizau/protocols/commit/6bf41ebb16f537ba317601fa741799eae672cc4a
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by ffflorian** [comment]
@peterblazejewicz Would you mind reviewing this one, too? 🙂

File: types/protocols/index.d.ts [hunk 1]
```diff
@@ -1,14 +1,16 @@
-// Type definitions for protocols 1.4
+// Type definitions for protocols 2.0
 // Project: https://github.com/IonicaBizau/protocols
-// Definitions by: Florian Keller <https://github.com/ffflorian>
+// Definitions by: Florian Imdahl <https://github.com/ffflorian>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+/// <reference types="node" />
```
Comment (peterblazejewicz) [inline types/protocols/index.d.ts:6]: this is a module (it has CJS export), so this should be an import for `URL` symbol drectly (e.g. `import { URL } from ...`

**Comment by ffflorian** [comment]
@peterblazejewicz Thanks, I added the import. But it was still necessary to keep the `/// <reference types="node" />` line.

**Review by peterblazejewicz — APPROVED** [review]

**Comment by ffflorian** [comment]
Ready to merge

**Pull request merged**